translations/round_structure.tex/pl.po

```diff
@@ -80,6 +80,27 @@ msgid ""
 "Keep playing new Rounds until any of the Scenario's ending conditions have been met.\n"
 "\n"
 msgstr ""
+"Rozgrywka podzielona jest na Rundy, podczas których każdy gracz wykonuje swoją Turę w kolejności zgodnej z ruchem wskazówek zegara, zaczynając od gracza rozpoczynającego grę.\n"
+"W swoich Turach gracze przemieszczają swoich \\hyperlink{Heroes}{Bohaterów} na Mapie Gry, budują nowe Budynki w swoich \\hyperlink{Town}{Miastach} i rekrutują \\hyperlink{Units}{Jednostki}, próbując spełnić warunek zwycięstwa Scenariusza.\\par\n"
+"\\textbf{Na początku każdej Rundy}, z wyjątkiem pierwszej, wykonaj następujące kroki :\n"
+"\\begin{itemize}\n"
+" \\item Obróć na aktywną stronę wszystkie użyte Żetony Budowy, Populacji i/lub Księgi Zaklęć.\n"
+" \\item Obróć na aktywną, zieloną stronę użyte Żetony Ruchu (PR).\n"
+" \\item Odzyskaj możliwość korzystania z Efektów Mistrzowskich  \\includesvg[height=10px]{\\svgs/expert.svg}.\n"
```

Comment:
Wprawdzie poprawnie, ale brzmi to trochę tak, jakby Kali zdawał egzamin z polskiego na obywatelstwo. @Baritoshi, masz może pomysł, jak to przełożyć?

Reply:
"Odzyskaj wszystkie Efekty Mistrzowskie" ?

Reply:
Mam wrażenie że trochę nie po polsku. Ciężko "odzyskać efekt".

Reply:
Może tak, i tego nie widać w tym tekście, ale zaraz za tym jest ikonka korony, która wyraźnie wskazuje, o co chodzi. Tutaj nie chodziłoby o słownikowy efekt, a o komponent gry.